tests/integration/workflows/nodejs_npm/test_nodejs_npm.py

```diff
@@ -315,6 +315,46 @@ def test_build_in_source_with_download_dependencies(self, runtime):
         output_files = set(os.listdir(self.artifacts_dir))
         self.assertEqual(expected_files, output_files)
 
+    @parameterized.expand([("nodejs12.x",), ("nodejs14.x",), ("nodejs16.x",), ("nodejs18.x",), ("nodejs20.x",)])
```

Comment:
Do we still need node12 here?

Reply:
I think we should leave it until it is been deprecated https://github.com/aws/aws-sam-cli/issues/6298